iModelJsNodeAddon/JsCloudSqlite.cpp

```diff
@@ -471,10 +471,18 @@ struct JsCloudContainer : CloudContainer, Napi::ObjectWrap<JsCloudContainer> {
 
     Napi::Value CleanDeletedBlocks(NapiInfoCR info) {
         RequireWriteLock();
-        OPTIONAL_ARGUMENT_INTEGER(0, nSeconds, (60*60)); // default to one hour
+
+        bool debugLogging = false;
+        int nSeconds = 3600; // default is 1 hour.
+        if (info[0].IsObject()) {

```

Comment:
If you want to allow an object as an argument, can you update the TS function signature?

Reply:
Sorry.. I forgot to push that change